routes/University/controllers.js

```diff
@@ -9,24 +9,62 @@ const router = express.Router();
 router.post('/', checkSchema(universitySchema), async (req, res, next) => {
    const {
       name,
-      city
+      shortname,
+      city,
+      description,
+      website,
+      year,
+      faculties,
+      students
    } = req.body;
 
    try {
-      await service.create(name, city);
+      await service.create(name, shortname, city,  description, website, year, faculties, students);
```

Comment:
I think it would be shorter to pass the entire body. Especially as there are no checks on the deconstructed properties.

Reply:
done